tools/swift-format/Sources/Rules/FullyIndirectEnum.swift

```diff
@@ -12,5 +13,67 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#enum-cases
 public final class FullyIndirectEnum: SyntaxFormatRule {
+  public override func visit(_ node: EnumDeclSyntax) -> DeclSyntax {
+    let enumMembers = node.members.members
+    
+    // Verifies all cases are indirect
+    var it = node.members.members.makeIterator()
+    while let caseMember = it.next()?.decl as? EnumCaseDeclSyntax {
+      guard let caseModifiers = caseMember.modifiers else { return node }
+      if isIndirectCase(modifiers: caseModifiers) { continue }
+      else { return node }
+    }
+    diagnose(.reassignIndirectKeyword(name: node.identifier.text), on: node.identifier)
+    
+    // Removes 'internal' keyword from cases, reformats
+    var newMembers: [MemberDeclListItemSyntax] = []
+    for member in enumMembers {
+      if let caseMember = member.decl as? EnumCaseDeclSyntax {
+        guard let caseModifiers = caseMember.modifiers else { continue }
+        let newCase = caseMember.withModifiers(removeInternalModifier(curModifiers: caseModifiers))
+        let formattedCase = formatCase(unformattedCase: newCase)
+        let newMember = SyntaxFactory.makeMemberDeclListItem(decl: formattedCase, semicolon: nil)
+        newMembers.append(newMember)
+      } else {
+        newMembers.append(member)
+      }
+    }
+
+    let newMemberBlock = SyntaxFactory.makeMemberDeclBlock(leftBrace: SyntaxFactory.makeLeftBraceToken(),
```

Comment:
Nitpick: can you put each of these parameters on their own line and ensure 100 columns per line?